addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animations.py

```diff
@@ -122,14 +122,15 @@ def __gather_animation(blender_action: bpy.types.Action,
         print_console("WARNING", "Animation '{}' could not be exported. Cause: {}".format(name, error))
         return None
 
-    if not animation.channels:
-        return None
-
-    # To allow reuse of samplers in one animation,
-    __link_samplers(animation, export_settings)
+    if animation.channels:
```

Comment:
linking sampler here, before the hook, make the hook data available different of what we have before. Not sure we can change this, now that people know (and use?) this hook.
Solution: create another hook (so one before, and one after the link_samplers process?)